src/EventListener/LocaleListener.php

```diff
@@ -66,10 +66,6 @@ private function getLocale(Request $request): string
             return $this->formatLocaleId($request->attributes->get('_locale'));
         }
 
-        if (null !== ($session = $request->getSession()) && $session->has('_locale')) {
```

Comment:
Removing this is wrong. But we should check `$session->isStarted()` and only execute the logic then.

Reply:
I don't see why the locale needs to be stored in the session, makes absolutely no sense to me.

Reply:
It's a topic to discuss. It is/was there so the locale is available before routing and security. I do agree that we should remove that, but there are a lot more places to handle.